packages/tempus/src/Tempus_AdjointSensitivityModelEvaluator_decl.hpp

```diff
@@ -28,7 +28,7 @@ namespace Tempus {
  * to a product vector formed by its columns.  This is the form of the adjoint
  * equations suitable for computing pseudotransientsensitivities of a response
  * function g(x(T),p) or adjoint sensitivities for a time-integrated response
- * function \int_0^T g(x(t),p).
+ * function int_0^T g(x(t),p).
```

Comment:
You really want to edit this entire comment to surround all the equations with `\f$` markers, and switch the equations to appropriate LaTeX.

Reply:
I'm not sure I necessarily agree with that.  While wrapping with \f$ makes the generated html documentation nicer, it makes the raw comments much harder to read (in my opinion).